dockerfiles/cli/cli.sh

```diff
@@ -1015,7 +1032,6 @@ cmd_version() {
   text "$CHE_PRODUCT_NAME:\n"
   text "  Version:      %s\n" $(get_installed_version)
   text "  Installed:    %s\n" $(get_installed_installdate)
-  text "  CLI version:  $CHE_CLI_VERSION\n"
```

Comment:
The cli and the product version are now identical because the cli is in a container.  It is logged in the line above.